SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialAuthorityTest/SpatialAuthorityTest.h

```diff
@@ -56,6 +56,10 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialAuthorityTest : public ASpatialFunct
 	UPROPERTY(EditAnywhere, Category = "Default")
 	FVector Server2Position;
 
+	// This needs to be a position on the border between Server 1 and Server 2.
+	UPROPERTY(EditAnywhere, Category = "Default")
+	FVector BorderPosition;
```

Comment:
I think this might potentially avoid the problem depending on the float precision steps - is it possible to make the position just be 0,0,0 exactly (without the potential for the editor to modify it) and configure the world around 2+4 zones?

Reply:
Changed from a 1x2 load balancing strategy to 2x2.
Fixed BorderPosition at (0,0,0) which is on the border between the 4 workers.